catalog/USGS/USGS_NLCD_RELEASES_2023_REL_RCMAP_V6_COVER.jsonnet

```diff
@@ -0,0 +1,369 @@
+              '369f00',
+              '349d00',
+              '339900',
+              '339900',
+              '2f9200',
+              '2d9100',
+              '2d8f00',
+              '2c8a00',
+              '2c8800',
+              '2c8500',
+              '2c8400',
+              '2b8200',
+              '297d00',
+              '297a00',
+              '297900',
+              '277700',
+              '247400',
+              '247000',
+              '29700f',
+              '2c6d1c',
+              '2d6d24',
+              '336d2d',
+              '366c39',
+              '376c44',
+              '396a4a',
+              '396a55',
+              '3a6a5f',
+              '3a696a',
+              '396774',
+              '3a6782',
+              '39668a',
+              '376292',
+              '34629f',
+              '2f62ac',
+              '2c5fb7',
+              '245ec4',
+              '1e5ed0',
+              '115cdd',
+              '005ae0',
+              '0057dd',
+              '0152d6',
+              '0151d0',
+              '014fcc',
+              '014ac4',
+              '0147bd',
+              '0144b8',
+              '0142b0',
+              '0141ac',
+              '013da7',
+              '013aa0',
+              '01399d',
+              '013693',
+              '013491',
+              '012f8a',
+              '012d85',
+              '012c82',
+              '01297a'
+            ],
+            bands: [
+              'rangeland_annual_herbaceous',
+            ],
+          },
+        },
+      },
+    ],
+    'eo:bands': [
+      {
+        name: 'rangeland_annual_herbaceous',
+        description:
+          'Percent of the pixel covered by annual herbaceous vegetation.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_bare_ground',
+        description: 'Percent of the pixel covered by bare ground.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_non_sagebrush_shrub',
+        description:
+          'Percent of the pixel covered by non-sagebrush shrubs.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_herbaceous',
+        description: 'Percent of the pixel covered by herbaceous vegetation.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_litter',
+        description: 'Percent of the pixel covered by litter.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_sagebrush',
+        description:
+          'Percent of the pixel covered by sagebrush (*Artemisia* spp).',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_shrub',
+        description: 'Percent of the pixel covered by shrub.',
+        'gee:units': units.percent,
+      },
+      {
+        name: 'rangeland_shrub_height',
+        description: 'Average height of shrubs in centimeters.',
+        'gee:units': units.centimeters,

```

Comment:
units.centimeter (to match https://github.com/google/earthengine-catalog/blob/main/catalog/units.libsonnet#L120)